src/foam/nanos/fs/fileDropZone/FileDropZone.js

```diff
@@ -207,7 +211,7 @@ foam.CLASS({
               .end();
             }))
         .end()
-        .start().addClass(this.myClass('caption-container')).hide(this.files$.map((v) => { return v.length > 0; }))
+        .start().show(this.showHelp$).addClass(this.myClass('caption-container')).hide(this.files$.map((v) => { return v.length > 0; }))
```

Comment:
Why do you have a show and hide on the same element?

Reply:
Show is if you want to see it in the first place (help details), and hide is there to hide it once a file is added.

Reply:
I think they will conflict and should be converted into either a single hide() or show() which combines the two values:
show(this.slot(this.function(showHelp, files) { return showHelp && files.length > 0; });